doc/lsst.cp.verify/using-cp-verify.rst

```diff
@@ -0,0 +1,81 @@
+
+These residual images are the main input to `CpVerifyBiasTask`, a subclass of `CpVerifyStatsTask`, along with the `IsrTask` task metadata (for historical reasons; this was used as the source of the empirical read noise estimate), and any ``isrStatistics`` results.  The parent class has a handful of other input connections, allowing catalog results to be verified, and uncorrected versions of the input exposure and catalog (which have not had the calibration applied).  These connections are discussed below as they apply to particular calibration products.
+
+For simple measurements, the `CpVerifyBiasConfig` defines a set of dictionary configuration entries that have string keys that correspond to a measurement name that will appear in the output results, with a value that maps in most cases maps to an `lsst.afw.math.statistics.Property` indicating what mathematical operation will be used to perform that measurement.  If these dictionary configurations have non-zero length, then they are used by the task to determine which methods will be used to do the measurement.  These configuration options include:
+
+* ``imageStatKeywords``:  these options define the tests that will be run on each amplifier segment in the image being processed.  All following ``ImageStatKeyword`` entries follow this same behavior.
+
+* ``unmaskedImageStatKeywords``:  as above, these options define the tests run ignoring all mask planes present.
+
+* ``uncorrectedImageStatKeywords``:  these options define the tests that will be run using the ``uncorrectedExp`` input connection.
+
+* ``crImageStatKeywords``:  these options define the tests that will be run after cosmic ray rejection has been applied.
+
+* ``normImageStatKeywords``:  these options define the tests that will be run after normalizing the image by the exposure time.
+
+* ``detectorStatKeywords``:  as all previous image analysis is done on a per-amplifier basis, these options define tests to be run on the full detector.  As these will nearly always depend on the calibration under consideration, there is no defined method in the base class, so subclasses will need to implement it.
+
+* ``metadataStatKeywords``:  these options define the tests that will be run on the input task_metadata input.  These tests need to be implemented by subclasses.
+
+* ``catalogStatKeywords``:  these options define the tests that will be run on the input catalog data.  As before, these tests need to be implemented by subclasses.
+
+The final two configuration options that control the behavior of the verification task are the ``useIsrStatistics`` configuration option, which defines if there are results from the ``isrStatistics`` input that should be considered, and the ``hasMatrixCatalog`` option, which indicates if a matrix catalog output will be constructed.
+
```

Comment:
Even after reading the following paragraph, I'm still not sure what a "matrix" is.  I think a concrete example of what a matrix quantity is would help a lot.

Reply:
Updated text in the `outputMatrix` description:

> An example use case for this kind of matrix catalog is the crosstalk coefficients, which would be represented with columns for source and target detector and amplifier, with the value column containing the coefficient at which the source pixels imprint onto the target pixels.